apisix/plugins/limit-conn.lua

```diff
@@ -19,18 +19,95 @@ local limit_conn = require("apisix.plugins.limit-conn.init")
 
 
 local plugin_name = "limit-conn"
+
+local policy_to_additional_properties = {
```

Comment:
I think this part of code is also duplicated, please look into how we can avoid code duplication.

Reply:
you also need to make `limit-count` use the code `utils/redis.lua` to avoid code duplication.

Reply:
> I think this part of code is also duplicated, please look into how we can avoid code duplication.

No, I don't think so. There may be some parameters in "local" that need to be customized. So this does not need to be made publicly available. I think it would be nice to have a template for everyone to use.

Reply:
done